spec/kittikrb/cursor_spec.rb

```diff
@@ -122,15 +122,219 @@ def current_rendered_buffer
+      end
+
+      describe '#left' do
+        it 'should properly set left offset position of cursor' do
+          expect(cursor.left).to be_instance_of described_class
+          expect(current_coords).to eq [-1, 0]
+        end
+      end
+
+      describe '#right' do
+        it 'should properly set right offset position of cursor' do
+          expect(cursor.right).to be_instance_of described_class
+          expect(current_coords).to eq [1, 0]
+        end
+      end
+    end
+
+    context 'erase methods' do
+      let(:width) { $stdout.winsize[1] }
+      let(:height) { $stdout.winsize[0] }
+
+      describe '#erase' do
+        it 'should erase(write whitespace) the specified region in TTY' do
+          expect($stdout).to receive(:write).exactly(6).times.with("\e\s")
+          expect(cursor.erase(1, 1, 3, 2)).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToEnd' do
+        it 'should erase(write whitespace) all chars to end current row ' \
+           'in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(width - 6 + 1).times
+            .with("\e\s")
+          expect(cursor.eraseToEnd).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToStart' do
+        it 'should erase(write whitespace) all chars to current position in ' \
+           'row in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(6).times.with("\e\s")
+          expect(cursor.eraseToStart).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToDown' do
+        it 'should erase(write whitespace) all chars to current position in ' \
+           'column in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(6 + 1).times.with("\e\s")
+          expect(cursor.eraseToDown).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToUp' do
+        it 'should erase(write whitespace) all chars to end current column ' \
+           'in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(height - 6).times
+            .with("\e\s")
+          expect(cursor.eraseToUp).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseLine' do
+        it 'should erase(write whitespace) all chars in current row in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(width).times.with("\e\s")
+          expect(cursor.eraseLine).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseScreen' do
+        it 'should erase(write whitespace) all chars in TTY' do
+          expect($stdout).to receive(:write).exactly(width * height).times
+            .with("\e\s")
+          expect(cursor.eraseScreen).to be_instance_of described_class
+        end
+      end
+    end
+
+    context 'cursor methods' do
```

Comment:
Every method in this class is the cursor method :). Name this section like 'display methods' or 'styling methods'.